packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewerContainer.spec.js

```diff
@@ -35,106 +36,95 @@ const failSubject = Factory.build('subject', {
 
 describe('Component > BarChartViewerContainer', function () {
   it('should render without crashing', function () {
-    const wrapper = shallow(<BarChartViewerContainer />, { disableLifecycleMethods: true })
+    const wrapper = mount(<BarChartViewerContainer />)
     expect(wrapper).to.be.ok()
   })
 
-  it('should initialize with the default state', function () {
-    const wrapper = shallow(
-      <BarChartViewerContainer />,
-      { disableLifecycleMethods: true }
-    )
-    const mockState = {
-      JSONdata: {}
-    }
-    expect(wrapper.state()).to.eql(mockState)
-  })
-
   describe('without a subject', function () {
     it('should render null with the default props', function () {
-      const wrapper = shallow(<BarChartViewerContainer />)
+      const wrapper = mount(<BarChartViewerContainer />)
       expect(wrapper.html()).to.be.null()
     })
   })
 
   describe('with an invalid subject', function () {
-    let cdmSpy
-    let onErrorSpy
     let nockScope
     let wrapper
     before(function () {
       sinon.stub(console, 'error')
-      cdmSpy = sinon.spy(BarChartViewerContainer.prototype, 'componentDidMount')
-      onErrorSpy = sinon.spy()
       nockScope = nock('http://localhost:8080')
```

Comment:
In the project app tests, we mock the Panoptes API with nock. Maybe that would be better for the subject viewer tests too?